app/docs/md/learn/concepts/styling/index.md

```diff
@@ -2,64 +2,45 @@
 title: Styling
 ---
 
-An important part of both user experience and developer experience is applying styles to your app.
-There are a lot of solutions for approaching CSS, but very few that are designed for both the user and the developer in mind.
+Since Enhance is standards based you are free to style however you like. You can add a stylesheet to the public folder and start cowboying styles like an absolute badass or bring whatever tooling you like as long as it can output a static CSS file to the public folder of your app you are good to go.
 
```

Comment:
I'd re-order the below to be:

1. Enhance Styles
2. Plain old CSS like we used to do, but better
3. Sass
4. Tailwind

Reply:
People want CSS first. Then we can ease them in.